crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI058.py

```diff
@@ -1,82 +1,174 @@
-import collections.abc
-import typing
-from collections.abc import AsyncGenerator, Generator
-from typing import Any
+def scope():
+    from collections.abc import Generator
```

Comment:
Unfortunately, I had to scope each test here into its own function so that they can reason about the imports independently.

Reply:
Ahh, that makes sense, was wondering if that would be an issue